subprojects/core/src/main/java/org/gradle/api/internal/tasks/SnapshotTaskInputsBuildOperationType.java

```diff
@@ -37,7 +43,8 @@
 
     @UsedByScanPlugin
     public interface Details {
-        SnapshotTaskInputsBuildOperationType.Details INSTANCE = new SnapshotTaskInputsBuildOperationType.Details() {};
+        SnapshotTaskInputsBuildOperationType.Details INSTANCE = new SnapshotTaskInputsBuildOperationType.Details() {
```

Comment:
Was this change unintentional?

Reply:
nope, it's my formatter in IDEA. Reverted